server/grpcservice/service.go

```diff
@@ -5,24 +5,32 @@ package grpcservice
 import (
 	"context"
 	"io"
+	"sync"
 	"time"
 
 	"github.com/CeresDB/ceresdbproto/pkg/metaservicepb"
 	"github.com/CeresDB/ceresmeta/pkg/log"
+	"github.com/CeresDB/ceresmeta/server/cluster"
+	"github.com/CeresDB/ceresmeta/server/member"
+	"github.com/pkg/errors"
 	"go.uber.org/zap"
 )
 
 type Service struct {
 	metaservicepb.UnimplementedCeresmetaRpcServiceServer
-
 	opTimeout time.Duration
 	h         Handler
+	manager   cluster.Manager
```

Comment:
The service seems should not hold the manager which should be held by `server`(`h` in this service).